bodhi/server/consumers/masher.py

```diff
@@ -1005,24 +1005,30 @@ def _get_master_repomd_url(self, arch):
             basestring: A URL on the master mirror where the repomd.xml file should be synchronized.
         """
         release = self.compose.release.id_prefix.lower().replace('-', '_')
+        version = self.compose.release.version
         request = self.compose.request.value
 
+        # First check to see if there's an override for the current version, if not, fall back.
+        # This will first try fedora_28_stable_(suffix), and then fedora_stable_(suffix).
+        key_prefixes = ['%s_%s_%s' % (release, version, request),
+                        '%s_%s' % (release, request)]
```

Comment:
Now is not the time for the following idea, but I really want to get this kinda stuff out of the settings file and onto the Release object. I really hate that we have settings where the name of the setting itself is dynamic. It's a nightmare to document :(

That said, this is in keeping with the status quo and things are broken today, and I don't know a better way to do it quickly.